packages/flutter_tools/lib/src/base/io.dart

```diff
@@ -48,11 +49,9 @@ import 'dart:io' as io
     stdout;
 
 import 'package:meta/meta.dart';
+import 'package:file/file.dart';
 
-import '../globals.dart' as globals;
 import 'async_guard.dart';
-import 'context.dart';
-import 'file_system.dart';
```

Comment:
Depend on package:file for the interface instead, so that we can migrate this without migrating file_system at the same time